tests/test_manylinux.py

```diff
@@ -195,7 +195,7 @@ def test_build_wheel_with_binary_executable(docker_container):
     policy, manylinux_id, python_id, io_folder = docker_container
     docker_exec(manylinux_id, 'yum install -y gsl-devel')
 
-    docker_exec(manylinux_id, ['bash', '-c', 'cd /auditwheel_src/tests/testpackage && python setup.py bdist_wheel -d /io'])
+    docker_exec(manylinux_id, ['bash', '-c', 'cd /auditwheel_src/tests/testpackage && python -m pip wheel -w /io .'])
```

Comment:
If you are going to use `pip` to build wheels, it's best to also use `--no-deps`.

I personally prefer `pep517.build`.

Reply:
As this is only invoked for for a test inside a container, I'd leave `pip` here to keep it simple. Adding `--no-deps`.